app/assets/javascripts/discourse/app/lib/sidebar/community-section/faq-section-link.js

```diff
@@ -0,0 +1,30 @@
+import I18n from "I18n";
+import { isEmpty } from "@ember/utils";
+
+import BaseSectionLink from "discourse/lib/sidebar/community-section/base-section-link";
+
+export default class FAQSectionLink extends BaseSectionLink {
+  get name() {
+    return "faq";
+  }
+
+  get route() {
+    if (isEmpty(this.siteSettings.faq_url)) {
+      return "faq";
+    }
+  }
+
+  get href() {
+    if (!isEmpty(this.siteSettings.faq_url)) {
```

Comment:
I wonder if we could just `return this.siteSettings.faq_url` here? Would it be `null` if it is not set? Since it says in the comment this will take precedence over `route`, then if this is null route could simply `return "faq";` and not even check for the URL

Reply:
O this is a very good point :+1: I can actually remove the conditionals here.